pkgs/tools/text/chroma/default.nix

```
@@ -1,16 +1,34 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, git }:
 
 buildGoModule rec {
   pname = "chroma";
-  version = "0.8.1";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner  = "alecthomas";
-    repo   = "chroma";
+    repo   = pname;
     rev    = "v${version}";
-    sha256 = "1gwwfn26aipzzvyy466gi6r54ypfy3ylnbi8c4xwch9pkgw16w98";
+    sha256 = "0vzxd0jvjaakwjvkkkjppakjb00z44k7gb5ng1i4924agh24n5ka";
+    leaveDotGit = true;
+    fetchSubmodules = true;
   };
 
+  nativeBuildInputs = [ git ];
+
+  # populate values otherwise taken care of by goreleaser
+  postPatch = ''
+    substituteInPlace cmd/chroma/main.go \
+      --replace 'version = "?"' 'version = "${version}"'
+
+    commit="$(git rev-parse HEAD)"
+    substituteInPlace cmd/chroma/main.go \
+      --replace 'commit  = "?"' "commit = \"$commit\""
+
+    date=$(git show -s --format=%aI "$commit")
+    substituteInPlace cmd/chroma/main.go \
+      --replace 'date    = "?"' "date = \"$date\""
```

```suggestion
    commit="$(git rev-parse HEAD)"
    date=$(git show -s --format=%aI "$commit")

    substituteInPlace cmd/chroma/main.go \
      --replace 'version = "?"' 'version = "${version}"' \
      --replace 'commit  = "?"' "commit = \"$commit\"" \
      --replace 'date    = "?"' "date = \"$date\""
```

Should have noticed that earlier. Sorry about that.

---

or I should've 🤦🏻. Fixed now.